app/jobs/regular/group_pm_alert.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Jobs
+  class GroupPmAlert < ::Jobs::Base
+    def execute(args)
+      user = User.find_by(id: args[:user_id])
+      group = Group.find_by(id: args[:group_id])
+      post = Post.find_by(id: args[:post_id])
+      topic = post.topic
```

Comment:
I like to exit early like this

```suggestion
      return unless user = User.find_by(id: args[:user_id])
      return unless group = Group.find_by(id: args[:group_id])
      return unless post = Post.find_by(id: args[:post_id])
      return unless topic = post.topic
```